var/spack/repos/builtin/packages/py-distributed/package.py

```diff
@@ -59,6 +60,7 @@ class PyDistributed(PythonPackage):
     depends_on("py-tblib@1.6.0:", type=("build", "run"), when="@2.11.0:")
     depends_on("py-toolz@0.7.4:", type=("build", "run"), when="@:2.12.0")
     depends_on("py-toolz@0.8.2:", type=("build", "run"), when="@2.13.0:")
+    depends_on("py-toolz@0.10.0:", type=("build", "run"), when="@2022.10.2:")
```

Comment:
toolz 0.8.2+ is still correct for this version, are you looking at master?

Reply:
That is a bug. See https://github.com/dask/distributed/pull/7309

Reply:
Good catch, can you add a comment with a link to this PR?